src/components/ToscaDataViewer/index.jsx

```diff
@@ -30,6 +30,8 @@ const ToscaDataViewer = (props) => {
   let browseUrl = res.urls || res.browse_urls;
   if (browseUrl) browseUrl = browseUrl.find((url) => url.startsWith("http"));
 
+  let hideBrowseLink = ((res.metadata && res.metadata.exists_in_object_store === false) ? true : false);
```

Comment:
@mkarim2017 remove the redundant parenthesis here
also use `const` instead of `let` because this variable will not be updated

```js
const hideBrowseLink = res.metadata && res.metadata.exists_in_object_store === false ? true : false;
```

Reply:
Done @DustinKLo